app/models/setting.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: settings
+#
+#  id    :integer          not null, primary key
+#  key   :string
```

Comment:
this is a key-value pair table that may be reusable for other settings, adding a table for only one record (min_version) was a big no, so this is the middle solution I think may be acceptable enough

Reply:
And why is that we want this to be a table in the database?

Reply:
The reason for this is to give a dynamically way update min_versions, if you can think of another way to add this with existing base code it would be great